src/standalone/Snow/Snow.jl

```diff
@@ -293,7 +293,11 @@ function SnowParameters{FT}(
     density::DM = MinimumDensityModel(FT(200)),
     z_0m = FT(0.0024),
     z_0b = FT(0.00024),
-    α_snow::AM = ConstantAlbedoModel(FT(0.8)),
```

Comment:
Some of our unit tests fail probably because they use this default.

Others fail probably because they are testing the default of the ZenithAngleAlbedo model and now that has changed.

can test locally with "julia --project=test test/standalone/Snow/test_file_name.jl"  for each of the files in that directory.